etc/commerce-to-commerce/1.11.0.0/map.xml.dist

```diff
@@ -3422,12 +3398,6 @@
             <ignore>
                 <datatype>catalog_product_bundle_price_index.customer_group_id</datatype>
             </ignore>
-            <ignore>
-                <datatype>catalog_product_index_price.customer_group_id</datatype>
-            </ignore>
-            <ignore>
-                <datatype>catalog_product_index_tier_price.customer_group_id</datatype>
-            </ignore>
```

Comment:
Unnecessary rules because the table is excluded by `catalog_product_index*` rule